src/scripts/html-transforms.js

```diff
@@ -1,11 +1,17 @@
+/**
+ * Takes two transforms returns a new function that takes an html document,
+ * runs the two transforms functions on it, and returns the the resulting document.
+ * @param {array} ...transforms The html document to transform.
+ * @returns {HTMLDocument}
```

Comment:
This should be a function, like so:

```suggestion
 * @returns {(document: HTMLDocument) => HTMLDocument}
```

Kind of like above, it returns a *function* that takes an `HTMLDocument` and returns an `HTMLDocument`.

Reply:
Another thing you can do to simplify a bit is to define a special type for the kind of function we keep using over and over again:

```js
/**
 * HtmlTransforms are functions that take an HTML Document and modify it in
 * some useful way, such as removing scripts.
 * @typedef {(document: HTMLDocument) => HTMLDocument} HtmlTransform
 */
```

Then you can use it as a shortcut later instead of writing out that whole definition:

```js
/**
 * Takes several transforms returns a new function that takes an html document,
 * runs the transform functions on it, and returns the the resulting document.
 * @param {...HtmlTransform} transforms The HTML transforms to combine.
 * @returns {HtmlTransform}
 */
export function compose (...transforms) {
  // Code goes here
}
```

An editor with support will then show you what an `HtmlTrasform` is:

<img width="703" alt="typedef" src="https://user-images.githubusercontent.com/74178/65796539-c3ffc380-e121-11e9-81a6-88f9b569c311.png">

…and what the right arguments for `compose()` are when you are typing it somewhere else:

<img width="881" alt="function-call" src="https://user-images.githubusercontent.com/74178/65796569-d417a300-e121-11e9-8b70-6f6a60280d80.png">

…and it’ll even help figure out what other variables based on that code are going to be:

<img width="571" alt="individual-variable" src="https://user-images.githubusercontent.com/74178/65796610-ea256380-e121-11e9-8324-b4f2237306af.png">